src/userSync.js

```diff
@@ -31,7 +36,7 @@ export function newUserSync(userSyncDependencies) {
 
   // for now - default both to false in case filterSettings config is absent/misconfigured
   let permittedPixels = {
-    image: false,
+    image: true,
```

Comment:
This was not mentioned in the PR,

Are we defaulting to enabling image pixels now?

Reply:
Hi, 

Yes, userSync feature had image pixels enabled by default, prior to this change. Reason for this change here is that, we removed certain deprecated fields, as a result, we lost that default setting. We're setting this value to true now to maintain the default setting.